templates/default/rabbitmq.config.erb

```diff
@@ -8,6 +8,10 @@
   ]},
 <% if node['rabbitmq']['web_console_ssl'] -%>
   {rabbitmq_management, [
+<% if node['rabbitmq']['management']['load_definitions'] %>
+    {load_definitions, "<%= node['rabbitmq']['management']['definitions_file'] %>"}
+    ]}
+<% end -%>
```

Comment:
This looks like it will close the `rabbitmq_management` config section too early.  If you've already tested these changes and it builds a valid config file, feel free to ignore this.

Reply:
Good catch. Should be corrected in https://github.com/rabbitmq/chef-cookbook/commit/2b6fb6f0788695ed170f2673c1649b17e7a857b1.

Reply:
Looks like 2b6fb6f0788695ed170f2673c1649b17e7a857b1 moves the `load_definitions` section under `rabbitmq_management -> listener`.  Can't wait for the new config file format  :)